README.md

```diff
@@ -35,12 +18,12 @@ The `.env` file is used by flask to set environment variables when running `flas
 
 ## Running the App
 
-Once the all dependencies have been installed, start the Flask app in development mode within the poetry environment by running:
+To start the vagrant virtual machine by running:
 ```bash
-$ poetry run flask run
+$ vagrant up --provision
```

Comment:
I would say there's no reason to insist the reader uses vagrant. They could still install poetry and python and run `poetry install` / `poetry run flask run` on their actual machine.

It should just be mentioned as an option. Also `--provision` isn't required - that's to force it to rerun any provisioning scripts. In fact, you generally do not want that.